tests/typescript_comments/mapped_types.ts

```diff
@@ -0,0 +1,28 @@
+type A = {
+  // commentA
+  [a in A]: string;
+}
```

Comment:
What about this?

```ts
type APrime = {
  [// commentAPrime
    a in A]: string
}
```

Reply:
**Prettier pr-4418**
[Playground link](https://deploy-preview-4418--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEMCeAHOACAggBQCcBLAWxwF5tgAdKbbAbQHpntJTzYCTy6GGAQ2zF6uALpJsAZxgkoAczoBfEABoQEDDGLRpyUIMKEIAd3xGE+lIIBuEYgBN1IAEaFBYANZwYAZQxPUQVkOQBXOA1RaThCGCJBBVJBZAAzQQAbGI0AK2kADwAhD29fP0FyABlRODTM7JBAwhjCZFRMOGkwEm0XDHkYAHUnGAALZAAOAAYNfogYwY8MNv7O2NtajUI4AEcw4m2EpJSkdKzIkBjSYlDCCI1pYIy4AEUwiHg6840YQVdhxxjZAAJh+HmIGWCAGEIJwTiAoNBNiAwjEACp-axnGLKZRAA)
```sh
--parser typescript
```

**Input:**
```tsx
type APrime = {
  [// commentAPrime
    a in A]: string
}
```

**Output:**
```tsx
type APrime = {
  // commentAPrime
  [a in A]: string
};

```